docs/60-schema-validation/2-validate-users.mdx

```diff
@@ -164,25 +183,23 @@ You need to run the script to apply the schema to the `users` collection.
 
 <TabItem value="java" label="☕️ Java Spring Boot">
 
-1. Stop the running app.
-    1. Locate the bottom panel and click on the `TERMINAL` tab.
-    1. Press Ctrl+C to interrup the running app
-
-1. Copy above code for `applySchemaValidation` as a method in the `LibraryApplication.java` class.
-1. Modify the `run` method to call `applySchemaValidation`
-    ```java
-    @Override
-    public void run(String... args) {
-        log.info("🚀 App Started");
-        applySchemaValidation();
-    }
-    ```
-1. Restart the app typing in the Terminal:
+The `SchemaValidationConfig` class is annotated with `@Configuration` and exposes an `@Bean ApplicationRunner`, 
+so the schema validator runs **automatically at startup**.
+
+1. Stop the running app (press **Ctrl+C** in the terminal where it’s running).
+
+2. Make sure you replaced the `// TODO` in `SchemaValidationConfig.java` with the provided schema validation code.
 
+3. Start the app again:
     ```bash
-    mvn spring-boot:start
+    mvn spring-boot:run
     ```
 
+When the application starts, you should see a log confirming that the validator was applied to the users collection, similar to:
```

Comment:
This will apply the schema every time the application starts. This will give the attendees the wrong impression. We should at least add comments explaining the they only need to do this once.